impeller/entity/geometry/point_field_geometry.cc

```diff
@@ -22,20 +22,16 @@ GeometryResult PointFieldGeometry::GetPositionBuffer(
   if (radius_ < 0.0) {
     return {};
   }
-  Matrix transform = entity.GetTransform();
-  Scalar determinant = transform.GetDeterminant();
-  if (determinant == 0) {
-    return {};
-  }
+  const Matrix& transform = entity.GetTransform();
 
-  Scalar min_size = 0.5f / sqrt(std::abs(determinant));
+  Scalar min_size = 0.5f / transform.GetMaxBasisLengthXY();
```

Comment:
We lost our protection against a zero scale here. Is it checked anywhere else?

Reply:
Added back.